### PR TITLE
Added isMarkLogic property & MarkLogic users are now represented by a red marker

### PR DESCRIPTION
**Pull request by liangdanica**
For issues #173, #175, #166

### CODE REVIEW

File: src/scripts/util.sjs [hunk 1]
```diff
@@ -9,6 +9,7 @@ var endpoint = keys.endpoint;
 var userID = keys.userID;
 var secretkey = keys.secretkey;
 
+xdmp.log("what's going on..");
```
Comment (kmcostel) [inline src/scripts/util.sjs]: Remove plz

File: src/scripts/util.sjs [hunk 2]
```diff
@@ -23,6 +24,12 @@ var getCoord = function(postalCode, country) {
     var point = res.toArray()[1].root.results[0].geometry.location;
     var lat = point.lat;
     var long = point.lng;
+
+    xdmp.log("hmm");
```
Comment (kmcostel) [inline src/scripts/util.sjs]: this too

File: src/scripts/util.sjs [hunk 3]
```diff
@@ -23,6 +24,12 @@ var getCoord = function(postalCode, country) {
...
+    xdmp.log(point);
+    xdmp.log(lat);
+    xdmp.log(long);
```
Comment (kmcostel) [inline src/scripts/util.sjs]: and these

File: src/scripts/updateData_REST.sjs [hunk 4]
```diff
@@ -92,7 +96,8 @@ try {
         if (!(EA.version in oldDoc.root.fullDetails.ea_version)) {
           json.fullDetails.ea_version.push(oldDoc.root.fullDetails.ea_version[0]);
         }
-
+        
+        xdmp.log('replacing document with: ' + JSON.stringify(json));
```
Comment (dmcassel) [inline src/scripts/updateData_REST.sjs]: please remove this line -- it's a lot of information to dump in the log after the debugging is done

**Comment by liangdanica** [comment]
Removed the lines. Thanks guys!

**Pull request merged**